mocha-multi.js

```diff
@@ -2,6 +2,7 @@ var fs = require('fs');
 var util = require('util');
 var debug = require('debug')('mocha:multi');
 var path = require('path');
+var isString = require('is-string');
```

Comment:
It's unlikely to cause issues if you leave it out, but I figured better safe than sorry.

```
> s = 's' ; typeof s
"string"
> s = String('s') ; typeof s
"string"
> s = new String('s') ; typeof s
"object"
```